package.json

```diff
@@ -8,7 +8,7 @@
     "/src",
     "!*.spec.ts"
   ],
-  "typings": "dist/js-client-sdk.d.ts",
+  "types": "dist/index.d.ts",
```

Comment:
`index.d.ts` is what is generated for our top-level `index.ts`

![image](https://github.com/Eppo-exp/js-client-sdk/assets/417605/462f52fb-ceb4-48fd-aae6-4bf1bced7250)